samples/openthread/ncp/README.rst

```diff
@@ -0,0 +1,88 @@
+.. _ot_ncp_sample:
+
+Thread: NCP sample
+##################
+
+The Thread NCP sample demonstrates the usage of OpenThread Network Co-Processor architecture inside the Zephyr shell.
+
+Overview
+********
+
+The sample demonstrates usage of NCP target on the MCU to communicate it with user WPAN Network Daemon, allowing to create for example Thread Border Router.
+It is important to mention that, according to NCP architecture, MCU part needs to cooperate with user higher layer process to altogether establish complete Thread Border Router.
+Recommended solution is to use docker version of OTBR (OpenThread Border Router) and WPAN Network Daemon tool (wpantund), working under Linux operating system, for such purpose (for more information go there: `OpenThread Border Router`_).
+
+There is possibility to configure number of OpenThread features, but by default application has COMMISSIONER, JOINER, BORDER_ROUTER, BORDER_AGENT and DHCP6_SERVER functionalities enabled.
```

Comment:
This needs to be rephrased.
Supported OT features are mentioned on this page: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_thread.html
If this sample does not support any of these features, we need to mention these unsupported features in a note.
If this sample supports features that are not on the list on the page I linked, then does this mean this sample introduces support for new features and they need to be added to that list?

Reply:
I think it does support all Thread features and does not introduce new one (at least for now). I just was basing on the NCS CLI sample, where also such things were written in the .rst file. If it is wrong, maybe it should be changed in both samples?